extensions/2.0/Khronos/KHR_materials_emissive_strength/README.md

```diff
@@ -0,0 +1,173 @@
+# KHR\_materials\_emissive\_strength
+
+## Contributors
+
+* Mike Bond, Adobe, [@miibond](https://github.com/MiiBond)
+* Alex Jamerson, Amazon
+* Thomas Dideriksen, Amazon
+* Alex Wood, Analytical Graphics, [@abwood](https://twitter.com/abwood)
+* Ed Mackey, Analytical Graphics, [@emackey](https://twitter.com/emackey)
+* Nicolas Savva, Autodesk [@nicolassavva-autodesk](https://github.com/nicolassavva-autodesk)
+* Henrik Edstrom, Autodesk
+* Tobias Haeussler, Dassault Systemes [@proog128](https://github.com/proog128)
+* Bastian Sdorra, Dassault Systemes [@bsdorra](https://github.com/bsdorra)
+* Emmett Lalish, Google [@elalish](https://github.com/elalish)
+* Alexey Knyazev, Individual Contributor, [@lexaknyazev](https://github.com/lexaknyazev)
+* Bruce Cherniak, Intel
+* Gary Hsu, Microsoft [@bghgary](https://twitter.com/bghgary)
+* Nicholas Barlow, Microsoft
+* Adam Morris, Target [@weegeekps](https://github.com/weegeekps)
+* Sandra Voelker, Target
+* Ben Houston, Threekit, [@bhouston](https://twitter.com/BenHouston3D)
+* Eric Chadwick, Wayfair [echadwick-wayfair](https://github.com/echadwick-wayfair)
+
+Copyright (C) 2021 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Exclusions
+
+* This extension must not be used on a material that also uses `KHR_materials_unlit`.
+
+## Overview
+
+The core glTF 2.0 material model includes an `emissiveFactor` and an `emissiveTexture` to control the color and
+intensity of the light being emitted by the material, clamped to the range [0.0, 1.0]. However, in PBR environments
+with high-dynamic range reflections and lighting, stronger emission effects may be desirable.
+
+In this extension, a new `emissiveStrength` scalar factor is supplied, that governs the upper limit of emissive
+strength per material.
+
+**Implementation Note**: This strength can be colored and tempered using the core material's `emissiveFactor`
+and `emissiveTexture` controls, permitting the strength to vary across the surface of the material.
+Supplying values above 1.0 for `emissiveStrength` can have an influence on
+reflections, tonemapping, blooming, and more.
+
+### Physical Units
+
+For implementations where a physical light unit is needed, the units for the multiplicative product of
+`emissiveStrength`, `emissiveFactor`, and `emissiveTexture` are candela per square
+meter (**cd / m<sup>2</sup>**), sometimes called *nits*.
+
+**Implementation Note**:
+Because the value is specified per square meter, it indicates the brightness of any given point along
+the surface.  However, the exact conversion from physical light units to the brightness of rendered
+pixels requires knowledge of the camera's exposure settings, which are left as an implementation
```

Comment:
I am not in favor of talking about camera exposure setting since there is no such thing in glTF.
I would like a mention that going from internal high range float values to those of the display is up to the implementation. 
This would usually involve some sort of tonemapping where the higher internal (float) range is converted to that of the output device.
This is needed regardless of camera exposure settings.